### PR TITLE
Switching to official grafana OpenAPI

### PR DESCRIPTION
**Pull request by safaci2000**

### CODE REVIEW

File: internal/config/config_test.go [hunk 1]
```diff
@@ -60,6 +60,7 @@ func TestSetup(t *testing.T) {
 }
 
 func TestWatchedFoldersConfig(t *testing.T) {
+	os.Setenv("GDG_CONTEXT_NAME", "qa")
```
Comment (daldoyle) [inline internal/config/config_test.go:63]: I think this line does nothing given the following block? Is this leftover testing?

File: internal/service/folders.go [hunk 2]
```diff
@@ -262,6 +264,7 @@ func (s *DashNGoImpl) UploadFolders(filter filters.Filter) []string {
 		}
 		params := folders.NewCreateFolderParams()
 		params.Body = &newFolder
+		//f, err := s.client.Folders.CreateFolder(params, s.getAuth())
```
Comment (daldoyle) [inline internal/service/folders.go:267]: This is the same as the following line unless I'm missing something.

File: internal/service/folders.go [hunk 3]
```diff
@@ -240,7 +240,9 @@ func (s *DashNGoImpl) UploadFolders(filter filters.Filter) []string {
 				continue
 			}
 		}
+
 		var newFolder models.CreateFolderCommand
```
Comment (daldoyle) [inline internal/service/folders.go:244]: This is the same as the following line unless I'm missing something.

**Review by daldoyle — COMMENTED** [review]
Overall looks good, a few cleanup items mentioned in here.

File: internal/service/folders.go (hunk 2 above)
Comment (safaci2000) [reply internal/service/folders.go:267]: Removed

File: internal/service/folders.go (hunk 3 above)
Comment (safaci2000) [reply internal/service/folders.go:244]: I'm not sure I follow, this declares a new variable that will be used below.

**Pull request merged**